compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/XCLBinGen.cpp

```diff
@@ -12,6 +12,7 @@
 
 #include "AMDAIETargets.h"
 #include "aie/Conversion/AIEVecToLLVM/AIEVecToLLVM.h"
```

Comment:
Here or later, we should absorb this change : https://github.com/Xilinx/mlir-aie/pull/1583 (related to https://github.com/nod-ai/iree-amd-aie/issues/467)

Reply:
Ya those changes should just be made on the files we have moved into iree-amd-aie. Not the scope of this PR though.